Automattic/jetpack
# wpcomsh: update path to Jetpack Color & Tonesque

Fixes https://github.com/Automattic/vulcan/issues/423

## Proposed changes:

This PR replaces Jetpack references to `class.color.php` and `tonesque.php` in the wpcomsh plugin with their equivalent in the `classic-theme-helper` package.

### Other information:


## Jetpack product discussion


Project thread: pfwV0U-e1-p2

## Does this pull request change what data or activity we track or use?


n/a

## Testing instructions:






I don't know of a way to test these changes don't break features. However, we can check they trigger no PHP fatals and that classes are available.

- Spin up a test WoA site
- Pull this branch locally, and build the plugin: `jetpack build plugins/wpcomsh`
- Sync the newly built plugin with the `mu-plugins/wpcomsh` folder in your test site, with the `jetpack rsync wpcomsh` command (you may want to make a backup of the folder first, to roll back once you've tested this PR).
- SSH to your site and open `wp-content/mu-plugins/wpcomsh-dev/functions.php`
- At the end of the file, add the following snippet:

```
require __DIR__ . '/lib/class.color.php';
error_log( class_exists( 'Jetpack_Color' ) );
```

- You shouldn't see any errors in your server logs and `class_exists` should return true
- Repeat the operation for Tonesque

```
require __DIR__ . '/lib/tonesque.php';
error_log( class_exists( 'Tonesque' ) );
```
